### PR TITLE
Email address save doc

### PR DESCRIPTION
**Pull request by Jakaboy3**
#1888
@yuli48

### CODE REVIEW

File: config/base.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 import re
 from django.core.exceptions import ImproperlyConfigured
 from django.db import connection
-
+import sys
```
Comment (DaleMcGrew) [inline config/base.py]: Thank you Yulisa and Jack! Could you please roll back (undo) your changes to the config/base.py file?

**Review by DaleMcGrew — CHANGES_REQUESTED** [review]
Thank you again Yulisa and Jack, could you also please roll back the changes to package-lock.json?

File: config/base.py (hunk 1 above)
Comment (DaleMcGrew) [reply config/base.py]: Could you also please roll back the changes to package-lock.json? Thank you in advance.

**Comment by Jakaboy3** [comment]
@DaleMcGrew We made another pull request with only one file since my partner and I do not know how to roll back the code.

**Pull request closed without merging**